pkgs/applications/networking/instant-messengers/poezio/default.nix

```diff
@@ -42,10 +42,14 @@ python3.pkgs.buildPythonApplication rec {
     "poezio"
   ];
 
+  preCheck = ''
+    rm -r poezio
+  '';
```

Comment:
why this?

Reply:
to prevent pytest import confusion. if the directory is not removed the import will try to use it rather than the built package which breaks the tests.

Reply:
added a comment.